### DISCUSSION TITLE
What do you use MsQuic for?

### DISCUSSION THREAD

**Discussion by nibanks** [microsoft/msquic#3354] [Show and tell]
Hi Folks!

We're trying to reach out and learn what everyone is using MsQuic for. What scenarios or protocols are you building on top? Client, server or both? What languages are you using? What kind of deployments might you have?

We're also looking for possible partners that could help us test new features (such as Hystart++ or ECN) in real deployments to get additional confidence.

We'd love you hear from you! Thanks!

**Comment by TanmayRajore** [comment]
Working on Multi-path Quic and right now creating a file transfer application which might have real time components. Would love to help you out with testing.
Please let me know how I can help.

**Reply by nibanks** [reply]
Sounds neat. We'd love to work with you! Are you trying to implement the QUIC multipath draft for MsQuic?
